doc/source/installation.rst

```diff
@@ -39,6 +39,13 @@ Pip
 
 :std:doc:`pip <pip:usage>` is the only supported installation method.
 
+Keep in mind that on selinux supporting systems, if you install into a virtual
+environment, you may face https://github.com/ansible/ansible/issues/34340 even
```

Comment:
You can use extlink ref here (https://github.com/ansible/molecule/blob/6f14f0e/doc/source/conf.py#L76):
```suggestion
environment, you may face :gh:`issue <ansible/ansible/issues/34340>` even
```